.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create an issue to track a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Description
+
+A clear and concise description of what the bug is.
+
+- Does it occur on any specific browsers or breakpoints?
+- Remember to assign this to one of our GitHub project boards and to a milestone if applicable
```

Comment:
Is it perhaps better for us to do the triage rather than expect others to know about our projects or milestones?

Reply:
I was actually expecting this template mostly to be used by us as a way of automating our own issue creation. Do you think there's a possibility that others will make regular use of it too?